app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -228,6 +229,7 @@ key --skip
 <% else -%>
 <%= @host.diskLayout %>
 <% end -%>
+<%= snippet('kickstart_rhsm') if rhel_compatible && os_major > 8 -%>
```

Comment:
Nit: I'm generally not a fan of `> x`. IMHO `>= (x+1)` is easier to read. It also brings consistency with the `else` branch, which is `< x`.
```suggestion
<%= snippet('kickstart_rhsm') if rhel_compatible && os_major >= 9 -%>
```